src/assets/i18n/en.json5

```diff
@@ -3644,6 +3644,26 @@
 
   "submission.sections.describe.relationship-lookup.external-source.import-button-title.isProjectOfPublication": "Project",
 
+  "submission.sections.describe.relationship-lookup.external-source.import-button-title.none = "Import remote item",
```

Comment:
@Micheleboychuk : This is the wrong format which is why all tests are failing. It's using an equals instead of a colon.  Should be updated to:
`"submission.sections.describe.relationship-lookup.external-source.import-button-title.none":  "Import remote item",`